.travis.yml

```diff
@@ -13,21 +13,24 @@ env:
     - DISTRIB="conda" PYTHON_VERSION="2.7"
       NUMPY_VERSION="1.8.1" SCIPY_VERSION="0.13.3"
       SCIKIT_LEARN_VERSION="0.14.1" MATPLOTLIB_VERSION="1.3.1"
+      PANDAS_VERSION="0.13.0"
     # Ubuntu 14.04 versions without matplotlib
     - DISTRIB="conda" PYTHON_VERSION="2.7"
       NUMPY_VERSION="1.8.1" SCIPY_VERSION="0.13.3"
       SCIKIT_LEARN_VERSION="0.14.1"
+      PANDAS_VERSION="0.13.0"
     - DISTRIB="neurodebian" PYTHON_VERSION="2.7"
     # Trying to get as close to the minimum required versions while
     # still having the package version available through conda
     - DISTRIB="conda" PYTHON_VERSION="2.6"
       NUMPY_VERSION="1.6.2" SCIPY_VERSION="0.11.0"
       SCIKIT_LEARN_VERSION="0.12.1" MATPLOTLIB_VERSION="1.1.1"
-      NIBABEL_VERSION="1.1.0"
+      NIBABEL_VERSION="1.1.0" PANDAS_VERSION="0.9.0"
```

Comment:
In my local tests, it seems you can get pandas 0.12 with numpy 1.6.2 through conda. Any reason why to use 0.9 here? Just asking because I seem to remember the minimum pandas required version was 0.14 previously.